### DISCUSSION TITLE
TypeError:  data is not iterable.

### DISCUSSION THREAD

**Discussion by ns-dev-code** [react-hook-form/react-hook-form#5321] [Q&A] [answered]
I am trying to use nested fields using useFieldArray.
While appending I am getting this error: [TypeError:  data is not iterable.]
This is happening only for the very first object in the field list.
My current state obj: 
const currentSelection = { 
...someState,
serviceRequest: [
   passenger:[ ],
 ]
}
I have 2 appends one for serviceRequest and other for passenger. 
serviceRequest append works fine.
Error happens in passenger append.

<img width="1440" alt="Screenshot 2021-05-25 at 10 58 02 AM" src="https://user-images.githubusercontent.com/73627685/119444002-29b5b000-bd48-11eb-88b0-028116bc5fdc.png">

**Comment by bluebill1049** [comment, marked as answer]
can you share a simple and reproduceable codesandbox?

**Reply by ns-dev-code** [reply]
Sure. I will.

**Reply by ns-dev-code** [reply]
@bluebill1049 it was a version issue. Just updated to latest resolved now.

**Reply by bluebill1049** [reply]
Awesome!
